docs/layers/hexagon-layer.md

```diff
@@ -131,15 +131,15 @@ Whether to enable cell elevation. Cell elevation scale by count of points in eac
 
 - Default: `100`
 
-Filter bins and re-calculate color by `upperPercentile`. Hexagons with counts
-bigger than the upperPercentile counts will be hidden.
+Filter bins and re-calculate color by `upperPercentile`. Hexagons with value
+larger than the upperPercentile will be hidden.
```

Comment:
> the larger bins that fall outside the percentile range will not have accurate color any more, so we have to hide i

I thought we could make the filtered out hexes a white transparent color - i.e. they would desaturate and just become ghosts, but could still retain their elevation and give the user a feeling for what he has filtered out.

> Also if we want to display transparency, we have to always enable depthTest. 

I guess this could be an optional feature. A prop...